Makefile

```diff
@@ -1,16 +1,16 @@
 .PHONY: docs
 
 install-dev:
-	pip3 install -e ".[dev,web,slack]"
+	pip3 install -e ".[dev,web,slack]" && cd ./examples/custom_materializations && pip3 install -e .
```

Comment:
this is kinda invasive, now i can't do pip install -e .dev anymore.

is there a way to add this into setup.py ?

Reply:
I couldn't find a non-hacky way to do this through `setup.py` unfortunately.

Reply:
is it something like this? https://stackoverflow.com/questions/35668295/how-to-include-and-install-local-dependencies-in-setup-py-in-python

Reply:
Yes, exactly. As far as I understand the `egg` package should already be available which doesn't solve the original problem, but I could be wrong and can try this approach out if you feel like it's better.

Reply:
there's gotta be a way to install from source

Reply:
I think I fixed this